File: src/main/java/dstu/csae/lab8/task1/task1.java
package dstu.csae.lab8.task1;

public class task1 {

    public static void main(String[] args) {
        boolean isJavaFun = true;
        boolean isKotlinBetter = false;

        System.out.println("Java is fun? " + isJavaFun);
        System.out.println("Kotlin is better? " + isKotlinBetter);

        boolean andResult = isJavaFun & isKotlinBetter;
        boolean orResult = isJavaFun | isKotlinBetter;
        boolean notResult = !isJavaFun;

        System.out.println("AND: " + andResult);
        System.out.println("OR: " + orResult);
        System.out.println("NOT: " + notResult);
    }

}

File: src/main/java/dstu/csae/lab8/task1/task1.kt
package dstu.csae.lab8.task1

fun main(){
    val isKotlinFun: Boolean = true
    val isJavaOld: Boolean = false

    println("Kotlin is fun? $isKotlinFun")
    println("Java is old? $isJavaOld")

    val andResult = isKotlinFun && isJavaOld
    val orResult = isKotlinFun || isJavaOld
    val notResult = !isKotlinFun

    println("AND: $andResult")
    println("OR: $orResult")
    println("NOT: $notResult")
}